test/metabase/api/search_test.clj

```diff
@@ -313,7 +313,7 @@
   (when (search/supports-index?)
     (testing "It can use an alternate search engine"
       (with-search-items-in-root-collection "test"
-        (let [resp (search-request :crowberto :q "test" :search_engine "fulltext")]
+        (let [resp (search-request :crowberto :q "test" :search_engine "fulltext" :limit 1)]
```

Comment:
When testing with a large dev appdb `search-request` tries to parse the result before the API has finished streaming. We don't care about the results, so we don't need any more results.